src/lib.rs

```diff
@@ -1367,7 +1367,14 @@ impl Build {
         if !msvc || !is_asm || !is_arm {
             cmd.arg("-c");
         }
-        if self.cuda && self.files.len() > 1 {
+        if self.cuda
+            && self
+                .files
+                .iter()
+                .filter(|file| file.extension() == Some(OsStr::new("cu")))
```

Comment:
Nit: can this be `.iter().any(|file| ...)` or is that prevented by MSRV.

Reply:
Documentation says "any() is short-circuiting," so one can't `.count()` it. One could use it in the second case, but since the suggestion is to make it a function for using in both cases, then it would have to be `.filter()`...

Reply:
Oh right, yeah, `>1` rather than `>=1`, my bad.